tutorial.md

```diff
@@ -132,7 +132,7 @@ Output:
 install -D -m 755 kpod /usr/bin/kpod
 install -D -m 755 ocid /usr/bin/ocid
 install -D -m 755 ocic /usr/bin/ocic
-install -D -m 755 conmon/conmon /usr/libexec/ocid/conmon
```

Comment:
If you are going to change the location of conmon, you would need to change all of the locations /usr/local/bin/ocid for example.  But again, the packaged by distribution locations will be in the standard paths.